github/github_test.go

```diff
@@ -105,6 +105,16 @@ func testHeader(t *testing.T, r *http.Request, header string, want string) {
 	}
 }
 
+func testCanonicalHeader(t *testing.T, r *http.Request, header string, want string) {
+	canonical := http.CanonicalHeaderKey(header)
+	for _, h := range r.Header[canonical] {
+		if h == want {
+			return
+		}
+	}
+	t.Errorf("Multiple header get, wanted %s but not found for header %s", want, header)
+}
```

Comment:
At quick glance, I still don't see why this method has multiple Accept headers.  We've never run into this problem before, so what's different here?

And if we do in fact need to support multiple values for a given header, I'd rather update `testHeader` to support that, which we can do in a way that doesn't require updating all the existing tests.  Something like:

```go
func testHeader(t *testing.T, r *http.Request, header string, want ...string) {
        got := r.Header[header]
        sort.Strings(got)
        sort.Strings(want)
        if !reflect.DeepEqual(got, want) {
                t.Errorf("Header.Get(%q) returned %s, want %s", header, got, want)
        }
}
```